src/Tribe/Aggregator/Service.php

```diff
@@ -526,6 +526,36 @@ public function post_import( $args ) {
 		return $this->post( 'import', $args );
 	}
 
+	public function update_import( $import_id, $args ) {
```

Comment:
:no_entry_sign: Missing doc comment for function update_import()

<sub>hash: acab1b7104194a15e5e4d15768cbb5d9</sub>